docs/notebooks/getting_started.ipynb

```diff
@@ -0,0 +1,1918 @@
+{
```

Comment:
<p>"Alternative" instead of transformed? Cos counts technically are not a transformation of the log-normalized data </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/scverse-tutorials/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>sounds good, changed!</p>